File: Modul 1/Soal 10/PRAK110-2410817210019-AchmadReihanAlfaiz.c
#include <stdio.h>
#include <math.h>

int main()
{
     float alas = 5.0, tinggi = 12.0;
     float sisi_b = sqrt(alas * alas + tinggi * tinggi);
     float luas = 0.5 * alas * tinggi;
     float keliling = alas + tinggi + sisi_b;

     printf("Diketahui : \nAlas = %.0f cm \nTinggi = %.0f cm \n", alas, tinggi);
     printf("\nJawab :\n");
     printf("Sisi A = %.0f cm\n", tinggi);
     printf("Sisi B = %.0f cm\n", sisi_b);
     printf("Sisi C = %.0f cm\n", alas);
     printf("Keliling = %.0f cm\n", keliling);
     printf("Luas = %.0f cm\n", luas);

     return 0;
}

File: Modul 1/Soal 10/PRAK110-2410817210019-AchmadReihanAlfaiz.py
from math import sqrt

alas = 5
tinggi = 12
sisi_b = sqrt(alas**2 + tinggi**2)
luas = 0.5 * alas * tinggi
keliling = alas + tinggi + sisi_b

print(f"Diketahui :\nAlas = {alas}\nTinggi = {tinggi}")
print("\nJawab :")
print(f"Sisi A = {tinggi} cm")
print(f"Sisi B = {sisi_b:.0f} cm")
print(f"Sisi C = {alas} cm")
print(f"Keliling = {keliling:.0f} cm")
print(f"Luas = {luas:.0f} cm")
